pyvista/utilities/geometric_objects.py

```diff
@@ -968,6 +970,13 @@ def Pyramid(points):
     >>> pyramid = pyvista.Pyramid([pointa, pointb, pointc, pointd, pointe])
     >>> pyramid.plot(show_edges=True, line_width=5)
     """
+    if points is None:
+        points = [[1.0, 1.0, 0.0],
+                  [-1.0, 1.0, 0.0],
+                  [-1.0, -1.0, 0.0],
+                  [1.0, -1.0, 0.0],
+                  [0.0, 0.0, 1.67]]
```

Comment:
What is the reason for making the height 1.67?

Reply:
This tends to make the pyramid look a bit more regular, which I needed in my example in
![image](https://user-images.githubusercontent.com/11981631/130181933-7df8b661-c4a8-46a6-96fd-9b0edda750f8.png)

We can change it back if you think it looks better otherwise.

Reply:
Great. I suggest to make it `np.sqrt(2.0**2-2.0*np.sqrt(2.0)/2.0) = 1.608...` to make that all the side length will be `2.0`.
```suggestion
                  [0.0, 0.0, np.sqrt(2.0**2-2.0*np.sqrt(2.0)/2.0)]]
```